packages/shared/src/interface/external/athenahealth/jwt-token.ts

```diff
@@ -1,4 +1,19 @@
+import z from "zod";
+
 export type AthenaJwtTokenData = {
   ah_practice: string;
   ah_department: string;
+  source: "athenahealth";
 };
```

Comment:
1. Why are those properties in snake_case? Are we storing exactly what we get from Athena? If we're parsing, it would be good to use our own naming pattern.
2. Why adding `source` here? The JwtTokenPerSource already has the source.

Reply:
From call: 

1. Came from Athena, will leave from now.
2. Adding source is hacky but for now, it's okay...